great-idea-website/css/index.css

```diff
@@ -63,4 +63,89 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+ body{
+	 padding:10px;
+ }
+
+ header nav{
+	 display:inline-block;
+	 padding: 10px 100px 0px 110px;
+	 /* border:1px solid red; */
+ }
+ header nav a{
+	 color:gray;
+	 margin: 20px;
+	 text-decoration:none;
+	 display:inline-block;
+
+ }
+
+ header img.logo{
+
+	display: inline-block;
+	margin-left:130px;
+
+ } 
+
+.top-left{
+	border-bottom: 1px solid black;
+}
+ .top-screen{
+
+	margin:150px;
+	
+}
+.top-screen h1{
+	font-size: 80px;
+	text-align: center;
+}
+button{
+	margin-left:110px;
+}
+
```

Comment:
Little optimisation here: you could get rid of the button margin and the text-align property for the h1 element. Then by adding the text-align property to the wrapping div element (`.top-screen`), it will center align both the text and the button together.